### PR TITLE
chore(ci): migrate deprecating set-output commands

### LINKED ISSUE #1438: CI: Update method of storing actions output
**What would you like to be added or enhanced**

Github started issuing warnings:

> **[performance-test-suite-detect-runners](https://github.com/codenotary/immudb/actions/runs/3330848279/jobs/5509867273#step:2:36)**
> The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
> 

**Why is this needed**

We should update out github actions to ensure those will work once Github Actions are disabling obsolete features.

### PR DESCRIPTION
**Pull request by miles170**
Replace deprecating `set-output` command with `$GITHUB_OUTPUT`.

Closes #1438.

### CODE REVIEW

**Review by byo — COMMENTED** [review]
Hi @miles170 , thanks for your contribution.

This PR is ready to be merged. We keep the git history linear thus if you'd like to keep your GPG signature then please rebase on the current master (IIRC merging with rebase through GithHub will use their own GPG signature).

P.S. Sorry it took so long, I was sure I already replied :facepalm:

**Comment by miles170** [comment]
> Hi @miles170 , thanks for your contribution.
> 
> This PR is ready to be merged. We keep the git history linear thus if you'd like to keep your GPG signature then please rebase on the current master (IIRC merging with rebase through GithHub will use their own GPG signature).
> 
> P.S. Sorry it took so long, I was sure I already replied 🤦

It seems that this PR is already based on the latest master.

**Comment by byo** [comment]
> It seems that this PR is already based on the latest master.

Can you double-check? Parent of the commit in this pr is different to what the current master is:

```
$ git cat-file -p 0ae1b8fcad90fd8e661af95e32e92d07b62f599a | grep parent
parent d20fe1166d90084fd551751d7bb28424cdb34e15
$ git rev-parse origin/master
c33e2db1fad946da705c101a7c3af75264f5917a
```

**Comment by miles170** [comment]
> > It seems that this PR is already based on the latest master.
> 
> Can you double-check? Parent of the commit in this pr is different to what the current master is:
> 
> ```
> $ git cat-file -p 0ae1b8fcad90fd8e661af95e32e92d07b62f599a | grep parent
> parent d20fe1166d90084fd551751d7bb28424cdb34e15
> $ git rev-parse origin/master
> c33e2db1fad946da705c101a7c3af75264f5917a
> ```

I'm really sorry. Please check again. Thanks!

**Review by byo — DISMISSED** [review]

**Comment by byo** [comment]
Hi @miles170 , the master moved again :/ can you please rebase once more? I'll merge the PR shortly after (I'm not sure if you'll be able to merge it yourself but please do if there's such option, I'll make sure that the PR is all green)

**Comment by miles170** [comment]
I can't merge myself since I don't have write access.

**Review by byo — APPROVED** [review]

**Comment by byo** [comment]
> I can't merge myself since I don't have write access.

Ok, I'll merge it then once all checks are green

**Pull request merged**
